stacker/tests/fixtures/mock_blueprints.py

```diff
@@ -129,6 +140,28 @@ def create_template(self):
         self.template.add_resource(WaitConditionHandle("Dummy2"))
 
 
+class Broken(Blueprint):
+    """
+    This blueprint deliberately fails validation, so that it can be used to
+    test re-creation of a failed stack
+    """
+    VARIABLES = {
+        "StringVariable": {
+            "type": str,
+            "default": ""}
+    }
+
+    def create_template(self):
+        t = self.template
+        t.add_resource(WaitConditionHandle("BrokenDummy"))
+        t.add_resource(WaitCondition(
```

Comment:
Can you add a comment on the specific part of this that fails, and why (I'm assuming Timeout?)

Reply:
Timeout has a maximum value, which `2 ** 32` should always exceed. I'll add a comment to explain that.

Reply:
Documented.